CONTRIBUTING.md

```diff
@@ -131,3 +131,25 @@ Here's a checklist for you about what makes Pull Requests to ReSwift shine:
 * You can propose PRs to merge with the `master` branch directly. We don't use any complex branching strategies.
 
 As a contributor, choose the "squash & merge" strategy to merge PRs with a single commit, keeping the commit history clean. (That's an upside of focused Pull Requests: you don't lose extra information.)
+
+## Publishing New Releases
+
+Currently @DivineDiminion, @mjarvis, @Ben-G and @agentk are folks that have the necessary permissions to publish a new version to CocoaPods. If you want a new version of ReSwift to be published you should ping these folks.
```

Comment:
I've added a new GitHub team for the organization named @ReSwift/releases -- That way if the team of people capable to release changes we don't need to update this document.

Reply:
Good idea 👍 Having a link to the team would be easier to maintain than editing the GitHub handles of users with the access rights. Also, I think we should recruit a 5th person because I'm not the best candidate for this as I don't even work with CocoaPods and have far less experience with all this infrastructure :)

Reply:
@DivineDominion No worries. If you're not comfortable with this, we're all happy with your current level of contributions. We're just saying that we trust you to have the permissions to do so if you chose to.

My desire is just making sure that we have enough people with permissions that, if a quick hotfix is needed, theres someone that can step up and do so without waiting weeks.

Reply:
Great idea, will link to the team instead!